packages/core/src/actors/callback.ts

```diff
@@ -29,7 +29,7 @@ export function fromCallback<TEvent extends EventObject>(
 
       return state;
     },
-    transition: (state, event, { self, id }) => {
+    transition: function callbackTransition(state, event, { self, id }) {
```

Comment:
I just did this as a test because it was an anonymous function in debugging; @Andarist any better ways to make this non-anonymous? Use a class?

Reply:
it should have an inferred name here:
```ts
var obj = {
  someFn: () => { throw new Error('test') }
}
obj.someFn()
// 17:12:58.428 VM328:2 Uncaught Error: test
//    at Object.someFn (<anonymous>:2:25)
//    at <anonymous>:4:5
```
which of course wouldn't exactly be entirely helpful here - the inferred name would be `transition` and not something like `callbackTransition`.

I don't think classes would help to solve this particular problem, the method name would still be pretty generic (`transition`). I think what you did here is OK - we had an anonymous function, and now it's named. Although I would probably try to stick to arrows... since arrows might play better with minifiers.

We have a lot of anonymous arrows like this in the codebase and this has been raised as an issue for the first time, so I wonder how big of a problem this truly is. 🤷‍♂️ 

Naming an arrow function would require hoisting it and assigning to a local variable.